bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/RefreshCapability.java

```diff
@@ -64,7 +65,7 @@ public void dispose() {
 
     @Override
     public void expireData() {
-        dataTimeStamp = ZonedDateTime.now().minus(dataValidity);
+        dataTimeStamp = Instant.now().minus(dataValidity);
```

Comment:
I cannot comment unchanged code, but method `dataAge()` below should now use `Instant.now()` to calculate duration. After this there should be no need for importing `java.time.ZonedDateTime`.

Reply:
Correct !